pkg/collector/python/init.go

```diff
@@ -122,7 +122,7 @@ void initAggregatorModule(rtloader_t *rtloader) {
 // _util module
 //
 
-void GetSubprocessOutput(char **, int, char **, char **, int*, char **);
+void GetSubprocessOutput(char **, char**, char **, char **, int*, char **);
```

Comment:
formatting nit, for consistency
```suggestion
void GetSubprocessOutput(char **, char **, char **, char **, int*, char **);
```